tests/raftstore/cluster.rs

```diff
@@ -458,6 +461,63 @@ impl<T: Simulator> Cluster<T> {
         assert!(status_resp.has_region_detail());
         status_resp.take_region_detail()
     }
+
+    pub fn partition(&mut self, s1: &HashSet<u64>, s2: &HashSet<u64>) {
+        for node_id in s1 {
+            let filter = new_partition_filter(s2);
+            self.sim.wl().push_trans_filter(*node_id, filter);
+        }
+        for node_id in s2 {
+            let filter = new_partition_filter(s1);
+            self.sim.wl().push_trans_filter(*node_id, filter);
+        }
+    }
+
+    pub fn heal_partition(&mut self, s1: &HashSet<u64>, s2: &HashSet<u64>) {
+        for node_id in s1 {
+            self.sim.wl().pop_trans_filter(*node_id);
```

Comment:
partition I provide is something related to implement, user should know it's based on filters. 
and calling partition&heal_partition symmetric is also user  guaranteed

maybe I should add some code comments